docker/quorum-k8s-hooks/Dockerfile

```diff
@@ -2,21 +2,18 @@ FROM debian:latest
 
 RUN set -eux; \
     mkdir -p /root/.kube/ ; \
-    #https://github.com/balena-io-library/base-images/issues/273
-    mkdir -p /usr/share/man/man1; \
     apt-get update && apt-get install -y --no-install-recommends \
-    pkg-config ca-certificates apt-transport-https gnupg gnupg2 lsb-release software-properties-common jq libc6-dev make curl wget vim unzip libsodium-dev; \
+    pkg-config ca-certificates apt-transport-https gnupg gnupg2 lsb-release software-properties-common jq libc6-dev make curl wget unzip libsodium-dev; \
     echo "fd97de6b91a121428112c52e5fe04a15" > /etc/machine-id ; \
-    # kubectl    
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" && chmod a+x ./kubectl && mv ./kubectl /usr/local/bin/kubectl ; \
-    # helm
-    wget -q https://get.helm.sh/helm-v3.8.0-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm && chmod +x /usr/local/bin/helm ;
+    # kubectl
+    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" && chmod a+x ./kubectl && mv ./kubectl /usr/local/bin/kubectl ;
 
 RUN set -eux; \
     # install nodejs 17.x
     curl -fsSL https://deb.nodesource.com/setup_17.x | bash - ; \
     apt-get install -y nodejs; \
-    npm install -g npm@latest;
+    npm install -g npm@latest; \
+    npm install -g quorum-genesis-tool@latest;
```

Comment:
Lock this to a version as well, so can use 0.1.6 https://www.npmjs.com/package/quorum-genesis-tool and we can update when we release new versions of the QGT

Reply:
will do